includes/class-sensei-admin.php

```diff
@@ -1838,6 +1838,14 @@ public function ajax_log_event() {
 		$event_name = $_REQUEST['event_name'];
 		$properties = isset( $_REQUEST['properties'] ) ? $_REQUEST['properties'] : [];
 
+		// Set the source to js-event.
+		add_filter(
+			'sensei_event_logging_source',
+			function() {
```

Comment:
I'm okay with using anonymous functions with filters in very limited circumstances (included this one), though I have seen some chatter that it might be added as a _warning_ in WPCS.

Reply:
Hmm ok. I feel like this use case is not problematic, because developers can effectively remove the filter by adding another one later in the queue. We could always add an ignore comment if we start getting that warning.

Or, we could avoid using anonymous functions across the board. I feel like that's going to add an annoying amount of bloat to this case, though.

Reply:
I agree. I think this is a good example of when we should be using them, but just wanted to note it as I think it is the first use of an anonymous function in Sensei.